config/jobs/kubernetes/sig-instrumentation/verify-govet-levee.yaml

```diff
@@ -2,7 +2,7 @@ presubmits:
   kubernetes/kubernetes:
   - name: pull-kubernetes-verify-govet-levee
     cluster: k8s-infra-prow-build
-    always_run: false
+    always_run: true
```

Comment:
this should actually be marked `optional: true` if it has not been always_run / we don't have data that this specific configuration passes reliably (sorry if this was miscommunicated but testing it once on one PR isn't really what we're aiming for there).

the reason is, switching this to always_run true / implicit optional: false will make it required for merge.
even if that's your intended end-goal, it should only be true once it's been running reliably. we usually soak for at least a week.

it can switch to non-optional later.
both need to be outside of the test-freeze window (could not have done this much sooner than now)

Reply:
It is marked `optional: true` a couple lines below (on line 9, specifically), but it's not apparent in the diff shown by GitHub. It was `optional: true` before I made the change to `always_run`.

Is this ok?

I agree that this should not block PRs for now. Blocking PRs would not be coherent with the Beta target, which states:

> Analysis runs as a non-blocking presubmit check

Reply:
thanks 😅